content/design-systems-for-developers/react/pt/conclusion.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Conclusão"
+tocTitle: "Conclusão"
+description: "Sistemas de design prósperos economizam tem e aumentam produtividade"
+---
+
+Estudos suportados por pesquisa sugerem que a reutilização de código podem gerar entre [42-81% de economização de tempo](https://www.researchgate.net/publication/3188437_Evaluating_Software_Reuse_Alternatives_A_Model_and_Its_Application_to_an_Industrial_Case_Study?ev=publicSearchHeader&_sg=g8WraNGZNGPw0R-1-jGpy0XwUDeAr3qb472J6lhisyQ3l24pSmndO6anMdX2L3HdWHifsczPegR9wjA) e aumentar a produtividade em [40%](http://www.cin.ufpe.br/~in1045/papers/art03.pdf). Não deverá ser surpresa então, que os sistemas de design que ajudam a facilitar a partilha de **código de interface de utilzador**, estão a ganhar uma popularidade crescente entre programadores.
```

Comment:
Can you update `utilzador` to `utilizador`?

Reply:
It was updated in the latest commit. Thank you for the feedback